packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1794,8 +1794,19 @@ class PointerScrollEvent extends PointerSignalEvent with _PointerEventDescriptio
     super.position,
     this.scrollDelta = Offset.zero,
     super.embedderId,
+    this.whenIgnored,
   });
 
+  /// Callback called when this event is [ignore]d.
```

Comment:
nit: s/when/if/ (this is because the timing is fixed; the caller does not get to choose when to call it; it has to be right away or it's too late)

Reply:
Yes, you're right.